mlflow/tensorflow/callback.py

```diff
@@ -0,0 +1,102 @@
+from tensorflow import keras
+
+from mlflow import log_param, log_text
+from mlflow.utils.autologging_utils import BatchMetricsLogger
+
+
+class MLflowLoggingCallback(keras.callbacks.Callback):
+    """Callback for logging Tensorflow training metrics to MLflow.
+
+    This callback logs training metrics every epoch or every n steps (defined by the user) to
+    MLflow.
+
+    Args:
+        run: an 'mlflow.entities.run.Run' instance, the MLflow run.
+        log_every_epoch: bool, If True, log metrics every epoch. If False, log metrics every n
```

Comment:
Can we use `log_every_n_epochs` instead?

Reply:
It's very rare that users have a callback working per N epochs, I think it makes more sense to just do every epoch.